### ISSUE TITLE
Multi-chain support?

### ISSUE THREAD

**Issue by mheinzinger** [aqlaboratory/openfold#358]
Hi, 
first of all: absolutely great work - thank you so much for putting so much time and effort into sharing the training code for SOTA structure prediction! :)
From my current understanding, all training is done on single chains. However, I would be really curious to play around finetuning the existing OpenFold model for protein-protein-interactions which would require inputting 2 or more chains. Did I potentially miss this feature? - If not: is there any plan from your end to include this functionality. - If not: could you give some pointers on what would need to be changed in order to get this working?
Thanks again, all the best,
Michael

**Comment by vaclavhanzl** [comment]
Hi Michael, maybe you want to explore the [multimer branch](https://github.com/aqlaboratory/openfold/tree/multimer).

**Comment by mheinzinger** [comment]
Thanks for the pointer - that is definitely extremely helpful!

I just wondered whether there is sort of a README or brief summary of the technical details or the bells&whistles that are supported for multimer-specific training? - From the OpenFold paper, I just found that they use the "AlphaFold-Gap implementation" and that "it falls short of the accuracy of AlphaFold-Multimer" but no comparison in terms of numbers (from the AF-multimer paper, Fig 1A, I read that this trick falls severely short in terms of performance). From [this line](https://github.com/aqlaboratory/openfold/blob/multimer/openfold/data/data_pipeline.py#L989), I also read that this is the current way of handling multiple chains.
